### DISCUSSION TITLE
how to set min.insync.replicas and replication factor

### DISCUSSION THREAD

**Discussion by ramandas30** [sngular/kloadgen#159] [Q&A] [answered]
Hi,
my scenario is i want to create topic having min.insync.replicas 2 and replication factor atlst 2.
there are 5 broker is running in my kafka cluster

**Comment by kszosze** [comment, marked as answer]
Hi @ramandas30,
Thanks for using the plugin and for the feedback. 
I'm gonna open an issue so we can add that property in the Sampler Producer.
Cheers!
